controllers/README.md

```diff
@@ -29,9 +29,9 @@ Create a `tilt-settings.json` file in this folder
 ```sh
 aws ecr-public get-login-password --region ${REGION} | docker login --username AWS --password-stdin public.ecr.aws/${REGISTRY_ALIAS}
 ```
-* `allowed_contexts`: list here the kube context of your cluster. By default, tilt won't interact with "non local" clusters and any eksa cluster, including the docker ones, are recognized as non local
+* `allowed_contexts`: list of the kube context of your cluster. By default, tilt won't interact with "nonlocal" clusters and any EKS-A cluster, including the Docker ones, are recognized as nonlocal
```

Comment:
Same as above. Tilt when not used as a folder name should be capitalized.

Reply:
Fixed in next revision